src/sentry/demo/data/scen1b/python_transaction.json

```diff
@@ -0,0 +1,165 @@
+{
+    "event_id": "2e75afe87658407cb6f11c65ed3d65fa",
+    "project": 5641789,
+    "release": "52f7c8a62a97cecf029f6a68adb411c1714c4836",
+    "dist": null,
+    "platform": "python",
+    "message": "",
+    "datetime": "2021-03-15T19:37:14.460185+00:00",
+    "tags": [
+      ["browser", "Safari 12.1.1"],
+      ["browser.name", "Safari"],
+      ["client_os", "Mac OS X 10.13.6"],
+      ["client_os.name", "Mac OS X"],
+      ["device", "Mac"],
+      ["device.family", "Mac"],
+      ["environment", "prod"],
+      ["http.status_code", "500"],
```

Comment:
Probably should change this to 200 since this transaction isn't erroring anymore

Reply:
@wmak good catch!